.github/workflows/node.js.yml

```
@@ -29,3 +29,23 @@ jobs:
     - run: npm ci
     - run: npm test
     - run: npm run eslint
+
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

would it be more useful to do (this is supposedly faster but I don't know what repercussions it could have) - 
```suggestion
          fetch-depth: 1
```

---

Should be 1 indeed!

---

Actually I am wrong, 0 is necessary. Will update the code.